### PR TITLE
build/1.82.0.2

### PR DESCRIPTION
**Pull request by MabezDev**

### CODE REVIEW

File: .github/workflows/publish-idf-rust-tags.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ on:
       toolchain-version:
         description: "Version of Rust toolchain"
         required: true
-        default: "1.80.0.0"
+        default: "v1.82.0.2"
```
Comment (SergioGasquez) [inline .github/workflows/publish-idf-rust-tags.yml:9]: I think we dont need the `v`
```suggestion
        default: "1.82.0.2"
```

File: .github/workflows/prepare-release.yaml [hunk 2]
```diff
@@ -6,11 +6,11 @@ on:
       last_release:
         description: "Previous Xtensa Rust release version"
         required: true
-        default: '1.80.0.0'
+        default: 'v1.82.0.2'
```
Comment (SergioGasquez) [inline .github/workflows/prepare-release.yaml:9]:
```suggestion
        default: '1.82.0.2'
```

File: .github/workflows/publish-idf-rust-tags.yml (hunk 1 above)
Comment (SergioGasquez) [reply .github/workflows/publish-idf-rust-tags.yml:9]: In this case, the `toolchain-version` argument is only used for the name of the tag, and all our previous tags dont have the `v`. We can start adding it, but people would be confused. 

https://hub.docker.com/r/espressif/idf-rust/tags

Comment (MabezDev) [reply .github/workflows/publish-idf-rust-tags.yml:9]: I can revert it for the docker action, but I think we need it for the rest - does that work for you?

Comment (SergioGasquez) [reply .github/workflows/publish-idf-rust-tags.yml:9]: Yes, we only need it without `v` in publish-idf-rust-tags.yml, the rest require it.

Comment (MabezDev) [reply .github/workflows/publish-idf-rust-tags.yml:9]: I applied this change to #278

**Pull request closed without merging**